MOBILE_API.xml

```diff
@@ -3699,6 +3709,417 @@
+            <description> 0 to 1, percentage humidity </description>
+        </param>
+        <param name="cloudCover" type="Float" mandatory="false">
+            <description> 0 to 1, percentage cloud cover </description>
+        </param>
+        <param name="moonPhase" type="Float" mandatory="false">
+            <description> 0 to 1, percentage of the moon seen, e.g. 0 = no moon, 0.25 = quarter moon </description>
+        </param>
+
+        <param name="windBearing" type="Integer" mandatory="false">
+            <description> In degrees, true north at 0 degrees </description>
+        </param>
+        <param name="windGust" type="Float" mandatory="false">
+            <description> km/hr </description>
+        </param>
+        <param name="windSpeed" type="Float" mandatory="false">
+            <description> km/hr </description>
+        </param>
+
+        <param name="nearestStormBearing" type="Integer" mandatory="false">
+            <description> In degrees, true north at 0 degrees </description>
+        </param>
+        <param name="nearestStormDistance" type="Integer" mandatory="false">
+            <description> In km </description>
+        </param>
+        <param name="precipAccumulation" type="Float" mandatory="false" >
+            <description> cm </description>
+        </param>
+        <param name="precipIntensity" type="Float" mandatory="false" >
+            <description> cm of water per hour </description>
+        </param>
+        <param name="precipProbability" type="Float" mandatory="false" >
+            <description> 0 to 1, percentage chance </description>
+        </param>
+        <param name="precipType" type="String" mandatory="false" >
+            <description> e.g. "rain", "snow", "sleet", "hail" </description>
+        </param>
+        <param name="visibility" type="Float" mandatory="false" >
+            <description> In km </description>
+        </param>
+
+        <param name="weatherIconImageName" type="String" mandatory="false" />
+    </struct>
+
+    <struct name="WeatherAlert">
+        <param name="title" type="String" mandatory="false" />
+        <param name="summary" type="String" mandatory="false" />
+        <param name="expires" type="DateTime" mandatory="false" />
+        <param name="regions" type="String" array="true" minSize="1" maxSize="99" mandatory="false" />
+        <param name="severity" type="String" mandatory="false" />
+        <param name="timeIssued" type="DateTime" mandatory="false" />
+    </struct>
+
+    <struct name="NavigationServiceManifest">
+		<param name="acceptsWayPoints" type="Boolean" mandatory="false">
+		<description>  Informs the subscriber if this service can actually accept way points.</description>
+		</param>
+	</struct>
+
+    <struct name="NavigationServiceData">
+	    <description> This data is related to what a navigation service would provide.</description>
+	
+	    <param name="timestamp" type="" mandatory="true">
```

Comment:
Type should be `DateTime`

Reply:
Param name should be `timeStamp`

Updates should look like: 
`<param name="timeStamp" type="DateTime" mandatory="true">
                <description> This is the timestamp of when the data was generated. This is to ensure any time or distance given in the data can accurately be adjusted if necessary. </description>`
            </param>